cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,636 @@
+        view.items_on_page.item_select("15")
+        view.search_box.fill("{}\n\n".format(self.name))
+        return (self.name in view.plans_completed_list.read() and
+                view.plans_completed_list.is_plan_succeeded(self.name))
+
+    def migration_plan_request(self):
+        view = navigate_to(self, "InProgress")
+        view.progress_card.select_plan(self.name)
+        request_view = self.create_view(MigrationPlanRequestDetailsView, wait="10s")
+        request_details_list = request_view.migration_request_details_list
+        view.items_on_page.item_select("15")
+        self.is_plan_in_progress()
+        return request_details_list
+
+
+@attr.s
+class MigrationPlanCollection(BaseCollection):
+    """Collection object for migration plan object"""
+
+    ENTITY = MigrationPlan
+
+    def create(
+        self,
+        name,
+        infra_map,
+        vm_list,
+        description=None,
+        csv_import=False,
+        target_provider=None,
+        osp_security_group=None,
+        osp_flavor=None,
+        pre_playbook=None,
+        post_playbook=None,
+        start_migration=False,
+    ):
+        """Create new migration plan in UI
+        Args:
+            name: (string) plan name
+            description: (string) plan description
+            infra_map: (object) infra map object name
+            vm_list: (list) list of vm objects
+            csv_import: (bool) flag for importing vms
+            target_provider:Target provider (OSP or RHV)
+            osp_security_group: security group for OSP
+            osp_flavor:Flavor for OSP,
+            pre_playbook: (string) pre-migration playbook name
+            post_playbook: (string) post-migration playbook name
+            start_migration: (bool) flag for start migration
+        """
+        view = navigate_to(self, "Add")
+
+        radio_btn = None
+        if csv_import:
+            radio_btn = "Import a CSV file with a list of VMs to be migrated"
+        view.general.fill({"infra_map": infra_map,
+                           "name": name,
+                           "description": description,
+                           "csv_import": radio_btn})
+
+        view.vms.wait_displayed()
+        view.vms.fill({
+            'csv_import': csv_import,
+            'vm_list': vm_list})
+
+        # For OSP we need to fill this extra tab
+        if target_provider:
+            if target_provider.one_of(OpenStackProvider):
+                view.instance_properties.wait_displayed()
+                view.instance_properties.fill({
+                    'osp_security_group': osp_security_group,
+                    'osp_flavor': osp_flavor
+                })
+
+        view.advanced.fill({
+            'pre_playbook': pre_playbook,
+            'post_playbook': post_playbook
+        })
+
+        view.schedule.fill(start_migration)
+        return self.instantiate(name=name, infra_map=infra_map, vm_list=vm_list)
+
+    def is_plan_started(self, name):
```

Comment:
Consider making this a property of the MigrationPlan entity, just like `in_progress`, `completed`, `successful`